routers/api/v1/admin/user.go

```diff
@@ -93,6 +93,13 @@ func CreateUser(ctx *context.APIContext) {
 	if ctx.Written() {
 		return
 	}
+
+	if u.LoginType == auth.Plain && len(form.Password) == 0 {
+		err := errors.New("PasswordIsRequired")
+		ctx.Error(http.StatusBadRequest, "PasswordIsRequired", err)
+		return
+	}
+
 	if !password.IsComplexEnough(form.Password) {
```

Comment:
Should this be changed according login type?

Reply:
Yes. We can refer to the following source code on the admin page.

https://github.com/go-gitea/gitea/blob/9a15267871dcb1f1f8b39115b8e68ed1f42bf21d/routers/web/admin/users.go#L154-L179

Reply:
@wolfogre See the commit [f9933a1](https://github.com/go-gitea/gitea/pull/28491/commits/f9933a18159c24d76036fab7e872794d43479437)

Reply:
I think it's not enough.

Reply:
@lunny See [dea9c50](https://github.com/go-gitea/gitea/pull/28491/commits/dea9c50baaac294a6c10ffd2317583980893c290)